insights/parsers/nftables.py

```diff
@@ -0,0 +1,154 @@
+                        "family": "ip", "name": "example_table1", "handle": 17
+                    }
+                },
+                {
+                    "map": {
+                        "family": "ip", "name": "example_map", "table": "example_table1", "type": "ipv4_addr", "handle": 2, "map": "verdict", "elem": [["192.0.2.1", {"accept": null}], ["192.0.2.2", {"drop": null}], ["192.0.2.3", {"accept": null}]]
+                    }
+                },
+                {
+                    "set": {
+                        "family": "ip", "name": "example_set", "table": "example_table1", "type": "ipv4_addr", "handle": 4
+                    }
+                },
+                {
+                    "chain": {
+                        "family": "ip", "table": "example_table1", "name": "example_chain1", "handle": 1, "type": "filter", "hook": "input", "prio": 0, "policy": "accept"
+                    }
+                },
+                {
+                    "rule": {
+                        "family": "ip", "table": "example_table1", "chain": "example_chain1", "handle": 3, "expr": [{"vmap": {"key": {"payload": {"protocol": "ip", "field": "saddr"}}, "data": "@example_map"}}]
+                    }
+                },
+                {
+                    "rule": {
+                        "family": "ip", "table": "example_table1", "chain": "example_chain1", "handle": 7, "expr": [{"match": {"op": "==", "left": {"payload": {"protocol": "tcp", "field": "dport"}}, "right": 22}}, {"counter": {"packets": 29, "bytes": 1976}}, {"accept": null}]
+                    }
+                }
+            ]
+        }
+
+    The json output is dict with a list of all the elements in nftables as the value of the "nftables" key.
+    To make it easier to use, here it extracts the address_family/table/chain/rule structures and
+    save it to main_data attribute.
+
+    The main_data attribute sample::
+
+        {
+            'ip': {
+                'example_table1': {
+                    'chains': {
+                        'example_chain1': {
+                            'attrs': {
+                                "family": "ip", "table": "example_table1", "name": "example_chain1", "handle": 1, "type": "filter", "hook": "input", "prio": 0, "policy": "accept"
+                            },
+                            'rules': [
+                                {
+                                    "family": "ip", "table": "example_table1", "chain": "example_chain1", "handle": 3, "expr": [{"vmap": {"key": {"payload": {"protocol": "ip", "field": "saddr"}}, "data": "@example_map"}}]
+                                },
+                                {
+                                    "family": "ip", "table": "example_table", "chain": "example_chain", "handle": 7, "expr": [{"match": {"op": "==", "left": {"payload": {"protocol": "tcp", "field": "dport"}}, "right": 22}}, {"counter": {"packets": 29, "bytes": 1976}}, {"accept": None}]
```

Comment:
For a prettier view, it will be better if we can split these too long lines into shorter ones, say line width <=80, <=100 ...